dm/tests/incremental_mode/conf/dm-task-no-gtid.yaml

```diff
@@ -0,0 +1,39 @@
+---
+name: test
+task-mode: all
```

Comment:
this is a `all` mode task, change to `incremental`? or move it into `tests/all_mode`